python-avd/pyavd/_eos_cli_config_gen/j2templates/eos/ethernet-interfaces.j2

```diff
@@ -685,6 +685,9 @@ interface {{ ethernet_interface.name }}
 {%     if ethernet_interface.ptp.enable is arista.avd.defined(true) %}
    ptp enable
 {%     endif %}
+{%     if ethernet_interface.ptp.profile.g8275_1.destination_mac_address is arista.avd.defined %}
```

Comment:
Config order is different from running config. Please move the logic after ptp delay-req interval. I know other options are not even in order but moving profile after delay-req interval would be better 
```
interface Ethernet1
   description P2P_LINK_TO_BORDER1-DC1_Ethernet3
   mtu 9214
   no switchport
   ip address 10.100.0.129/31
   ptp enable
   ptp announce interval 3
   ptp announce timeout 9
   ptp delay-req interval -7
   ptp profile g8275.1 destination mac-address forwardable
   ptp sync-message interval 1
```
Same for port-channel